client/src/main/java/org/eclipse/hono/client/HonoClient.java

```diff
@@ -338,6 +338,13 @@
     Future<MessageConsumer> createEventConsumer(String tenantId, BiConsumer<ProtonDelivery, Message> eventConsumer,
             Handler<Void> closeHandler);
 
+    Future<MessageConsumer> createCommandResponseConsumer(String tenantId, String replyId, Consumer<Message> commandResponseConsumer,
```

Comment:
*createAsyncCommandResponseConsumer*?

Reply:
I am fine with the naming :)

Reply:
this should be removed, shouldn't it?

Reply:
ups, so sorry. Something went terribly wrong with my local merging.